packages/common/src/services/ApiService.ts

```diff
@@ -246,13 +246,16 @@ export default class ApiService {
     return (await getDataOrThrow(response)) as AdSchedule;
   };
 
-  getMediaAds = async (url: string, mediaId: string): Promise<AdSchedule | undefined> => {
-    const urlWithQuery = createURL(url, {
-      media_id: mediaId,
+  getAppContentSearch = async (siteId: string, searchQuery: string | undefined) => {
```

Comment:
Did you intentionally replace the getMediaAds logic, or was this maybe a merge issue?

Reply:
Yeah, we don't use this endpoint anymore (we get ad config url from the app config).